### PR TITLE
feat(docs): add accessibility section for HueSlider component

### PR DESCRIPTION
**Pull request by chida09**
Closes https://github.com/yamada-ui/yamada-ui/issues/3329

## Description

スクリーンショット 2024-11-07 23 01 57

スクリーンショット 2024-11-07 23 02 06

### CODE REVIEW

**Review by 108yen — CHANGES_REQUESTED** [review]
Please unify the wording with https://github.com/yamada-ui/yamada-ui/pull/3322.

File: docs/contents/components/forms/hue-slider/accessibility.ja.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+| `End`        | スライダーの最大値に移動します。                | -    |
+| `Home`       | スライダーの最小値に移動します。                | -    |
+| `PageDown`   | スライダーの値を大きく減少(全体の10%分)します。 | -    |
+| `PageUp`     | スライダーの値を大きく増加(全体の10%分)します。 | -    |
+
+## ARIAロールと属性
+
+| コンポーネント | ロールと属性    | 使い方                                                                                                        |
+| -------------- | --------------- | ------------------------------------------------------------------------------------------------------------- |
+| Slider         | `role="slider"` | スライダーであることを示します。                                                                              |
```
Comment (108yen) [inline docs/contents/components/forms/hue-slider/accessibility.ja.mdx:33]: Change `Slider` to `SliderThumb`.
`SliderThumb` will separate by https://github.com/yamada-ui/yamada-ui/issues/3338.

File: docs/contents/components/forms/hue-slider/accessibility.ja.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+order: 4
+with_description: true
+tab: アクセシビリティ
+---
+
+`HueSlider`に`aria-label`を設定すると、スクリーンリーダーによって読み上げられます。
+
+```tsx live=false
+<HueSlider aria-label="Hue slider" />
+```
```
Comment (108yen) [inline docs/contents/components/forms/hue-slider/accessibility.ja.mdx:14]: It is not read by screen reader.

File: docs/contents/components/forms/hue-slider/accessibility.ja.mdx (hunk 1 above)
Comment (108yen) [inline docs/contents/components/forms/hue-slider/accessibility.ja.mdx:33]: Please add `aria-label`.

**Review by 108yen — CHANGES_REQUESTED** [review]

File: docs/contents/components/forms/hue-slider/accessibility.ja.mdx (hunk 2 above)
Comment (chida09) [reply docs/contents/components/forms/hue-slider/accessibility.ja.mdx:14]: What did you use to check this? 
I'd like to understand better.

File: docs/contents/components/forms/hue-slider/accessibility.ja.mdx (hunk 1 above)
Comment (108yen) [reply docs/contents/components/forms/hue-slider/accessibility.ja.mdx]: You probably misunderstand.
You don't neet to change role. Please change `Slider` of `Component`.

File: docs/contents/components/forms/hue-slider/accessibility.ja.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+| キー         | 説明                                            | 状態 |
+| ------------ | ----------------------------------------------- | ---- |
+| `ArrowDown`  | スライダーの値を1ステップ分減少します。         | -    |
+| `ArrowLeft`  | スライダーの値を1ステップ分減少します。         | -    |
+| `ArrowRight` | スライダーの値を1ステップ分増加します。         | -    |
+| `ArrowUp`    | スライダーの値を1ステップ分増加します。         | -    |
+| `End`        | スライダーの最大値に移動します。                | -    |
+| `Home`       | スライダーの最小値に移動します。                | -    |
+| `PageDown`   | スライダーの値を大きく減少(全体の10%分)します。 | -    |
+| `PageUp`     | スライダーの値を大きく増加(全体の10%分)します。 | -    |
```
Comment (108yen) [inline docs/contents/components/forms/hue-slider/accessibility.ja.mdx:21]: Please unify the wording of this section with https://github.com/yamada-ui/yamada-ui/pull/3322 too.

**Review by 108yen — CHANGES_REQUESTED** [review]
Is not `HueSlider` follows the [WAI-ARIA - Slider Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/slider/)?

File: docs/contents/components/forms/hue-slider/accessibility.ja.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+title: HueSlider
+description: "`HueSlider`は、ユーザーが色相を選択するために使用されるコンポーネントです。"
+package_name: "@yamada-ui/color-picker"
+order: 4
```
Comment (108yen) [inline docs/contents/components/forms/hue-slider/accessibility.ja.mdx:5]: `order: 3`?

File: docs/contents/components/forms/hue-slider/accessibility.ja.mdx (hunk 3 above)
Comment (chida09) [reply docs/contents/components/forms/hue-slider/accessibility.ja.mdx]: fixed.
https://github.com/yamada-ui/yamada-ui/pull/3337/commits/651e4d1957f2037f6df13634a7ce1ddab3beae60

**Comment by 108yen** [comment]
> Is not `HueSlider` follows the [WAI-ARIA - Slider Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/slider/)?

Have you checked this?

**Comment by chida09** [comment]
@108yen 
"I'm sorry. I don't understand what you want to say.

**Comment by 108yen** [comment]
> @108yen I'm sorry. I don't understand what you want to say.

If component follows WAI-ARIA pattern, we should indicate that in document.

For example, in the `Button' component, this component follows the [WAI-ARIA - Button Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/button/). So indicate in document like this.
`The Button follows the [WAI-ARIA - Button Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/button/) for accessibility.`
https://yamada-ui.com/components/forms/button/accessibility

**Comment by chida09** [comment]
@108yen 
Adding references with links to the official WAI-ARIA specifications.
https://github.com/yamada-ui/yamada-ui/pull/3337/commits/f9aa880cd5f94e235d2e7b1fd2a96db6141ce50d

File: docs/contents/components/forms/hue-slider/accessibility.ja.mdx [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+title: HueSlider
+description: "`HueSlider`は、ユーザーが色相を選択するために使用されるコンポーネントです。"
+package_name: "@yamada-ui/color-picker"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+`Slider`は、アクセシビリティに関して[WAI-ARIA - Slider Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/slider/)に従います。
```
Comment (108yen) [inline docs/contents/components/forms/hue-slider/accessibility.ja.mdx:10]: This is `HueSlider`.

**Review by 108yen — CHANGES_REQUESTED** [review]

**Review by 108yen — DISMISSED** [review]

**Comment by chida09** [comment]
I matched/aligned the nuances of the wording.
ref: https://github.com/yamada-ui/yamada-ui/pull/3322

**Review by 108yen — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**